yt/yt/server/lib/chunk_pools/ordered_chunk_pool.cpp

```diff
@@ -484,13 +486,16 @@ class TOrderedChunkPool
         int chunkSliceIndex = 0;
         while (chunkSliceIndex < std::ssize(chunkSlices) || pocket) {
             auto chunkSlice = pocket ? pocket : chunkSlices[chunkSliceIndex];
-            pocket = nullptr;
 
             if (!chunkSlice->GetRowCount()) {
+                // No empty pockets!
+                YT_VERIFY(!pocket);
```

Comment:
The comment says otherwise.

Reply:
Intended meaning: the slice that is saved in `pocket` cannot have zero rows.
It is just a consequence of the logic below and for this reason we do not need to write `if (!pocket) ++chunkSliceIndex`.

I will reword this comment.